Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
Date: 2018-10-22
Initial Package Version: 2.3.0
Upstream Status: Unknown
Origin: Found at Arch
Description: If building with static libs disabled, without this patch the
shared lib is not installed.

diff -rupN openjpeg-2.3.0/src/lib/openjp2/CMakeLists.txt openjpeg-2.3.0-new/src/lib/openjp2/CMakeLists.txt
--- openjpeg-2.3.0/src/lib/openjp2/CMakeLists.txt	2017-10-05 00:23:14.000000000 +0200
+++ openjpeg-2.3.0-new/src/lib/openjp2/CMakeLists.txt	2017-12-25 13:53:07.000000000 +0100
@@ -99,6 +99,7 @@ else()
     set(INSTALL_LIBS ${OPENJPEG_LIBRARY_NAME} openjp2_static)
   else()
     add_library(${OPENJPEG_LIBRARY_NAME} ${OPENJPEG_SRCS})
+    set(INSTALL_LIBS ${OPENJPEG_LIBRARY_NAME})
   endif()
 endif()
 

